modules/ROOT/Edits/modify-partner-settings.adoc

```diff
@@ -0,0 +1,62 @@
+= Modify a Partner's Settings
+
+You can modify a partner's settings including contact details, identifiers, digital certificates, acknowledgements, control numbers, and more.
+
+To modify a partner's settings:
+
+. In Partner Manager, select *Partners* in the navigation menu.
+. Click the partner.
```

Comment:
Compare with earlier wordings:

- Select the partner for whom to configure the send settings.
- Select the partner to associate with the settings.
- Select the partner for whom you want to configure the settings.

I know it is tedious, but if you have time, review the words used for repeated instructions and try to standardize them. I wouldn't even notice if I did not have so much of this text at one time...and probably the audience will not. But it is still a good goal to aspire to. :-)